admin/src/scenes/centersV2/components/sessions/SessionList.jsx

```diff
@@ -270,6 +277,23 @@ export default function SessionList({ center, sessions, user, focusedSession, on
                   </div>
                 </div>
               </div>
+              {center?._id && focusedSession?._id && center.region === "Provence-Alpes-Côte d'Azur" && (
```

Comment:
Du coup on met pas un flag en DB ?

Reply:
as you wish mais j'ai peur que ce flag ne servent qu'une fois et pourisse un peu la collection qui est déjà pas mal legacy

Reply:
non t'as raison
(il faudra trouver une meilleure manière de faire du feature flag à l'avenir)